c31-shadowlyn/react-nutshell-shadowlyn
# Cm friends

# Description

This branch contains functionality for the friends component. Also the registration form is included with this merge request.

Applies to issue # 6 / 8

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout cm-friends`
3. `npm start`
4. Test registration form by trying to register with your username and password. Then actually register a new account and make sure it takes you to the app and POSTS a new user
5. Look at the friends section. Test functionality of the search as well as the delete.

# Checklist:
